app/jobs/conversations/status_change_job.rb

```diff
@@ -0,0 +1,8 @@
+class Conversations::StatusChangeJob < ApplicationJob
+  queue_as :medium
+
+  def perform(account:, conversation_ids:, status:)
+    conversations = account.conversations.where(display_id: conversation_ids)
+    conversations.update(status: status)
```

Comment:
@tejaswinichile Would it show an activity message that the "conversation was resolved by X" if we do it like this?

Reply:
Yes, to create the activity we have `after_commit` assert actions set on the conversation model.